### DISCUSSION TITLE
S3Error: Access Denied.

### DISCUSSION THREAD

**Discussion by UmangSharma5** [minio/minio-js#1183] [Q&A]
`import * as Minio from 'minio';

const minioClient = new Minio.Client({
    // endPoint: 'play.min.io',
    // port: 9000,
    // useSSL: true,
    // accessKey: 'Q3AM3UQ867SPQQA43P2F',
    // secretKey: 'zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG'
    "endPoint":"10.8.0.31",
    "port":9000,
    "accessKey":"............",
    "secretKey":".......................",
    useSSL:false,
    "api":"s3v4",
    "path":"auto",
    
});

export {
    minioClient
}`

I have changed my minioClient from default to the one I own ([GNU AGPL v3]) and have created keys and updated my code as above with correct access and secret key.But I get an error Error creating bucket: S3Error: Access Denied.Can you suggest what might be going wrong

**Comment by prakashsvmx** [comment]
Do you have any policies attached to the user ?

You could run the trace and check. 
`mc admin trace <alias> -v` in a new terminal .

**Comment by prakashsvmx** [comment]
Feel free to open an issue if incorrect behavior is observed.
